### ISSUE TITLE
Connot convert BpmnModel to bpmn2.0 XML

### ISSUE THREAD

**Issue by ityelang** [Activiti/Activiti#3763]
**I try to convert a BpmnModel  to xml, but the result is

```xml
<?xml version="1.0" encoding="UTF-8"?>
<definitions ****>
  <process id="Lv" name="Lv" isExecutable="true">
    <startEvent id="start_event" name="startEvent">
      <outgoing>flow_0</outgoing>
    </startEvent>
```
**what I want is this version**

```xml
<?xml version="1.0" encoding="UTF-8"?>
<bpmn2:definitions 
xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
xmlns:bpmn2="http://www.omg.org/spec/BPMN/20100524/MODEL" 
xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" 
xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" 
id="sample-diagram" targetNamespace="http://activiti.org/bpmn" 
xsi:schemaLocation="http://www.omg.org/spec/BPMN/20100524/MODEL BPMN20.xsd">
  <bpmn2:process id="Process_1" isExecutable="true">
    <bpmn2:startEvent id="StartEvent_1" />
  </bpmn2:process>
  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
    <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="Process_1">
      <bpmndi:BPMNShape id="_BPMNShape_StartEvent_2" bpmnElement="StartEvent_1">
        <dc:Bounds x="412" y="240" width="36" height="36" />
      </bpmndi:BPMNShape>
    </bpmndi:BPMNPlane>
  </bpmndi:BPMNDiagram>
</bpmn2:definitions>
```
I found the source code already set default namespace to BPMN2.0, but it does not work.
```java
public static void writeRootElement(BpmnModel model, XMLStreamWriter xtw, String encoding) throws Exception {
    xtw.writeStartDocument(encoding, "1.0");

    // start definitions root element
    xtw.writeStartElement(ELEMENT_DEFINITIONS);
    xtw.setDefaultNamespace(BPMN2_NAMESPACE);
    xtw.writeDefaultNamespace(BPMN2_NAMESPACE);
```

my activiti version is 7.1.0.M4

**Comment by ityelang** [comment]
I found this commit resolved this issue ,but do not know the release version

https://github.com/Activiti/Activiti/commit/7c770c2fda0732df5db40d307daa3f0e05c95b7a

**Comment by ityelang** [comment]
<dependency>
  <groupId>org.activiti</groupId>
  <artifactId>activiti-spring-boot-starter</artifactId>
  <version>7.1.0.M4</version>
</dependency>

**Comment by ityelang** [comment]
I resolved this issue by upgrade version to 7.1.0.M6

**Comment by ityelang** [comment]
```java
public static void writeRootElement(BpmnModel model, XMLStreamWriter xtw, String encoding) throws Exception {
        xtw.writeStartDocument(encoding, "1.0");
        xtw.writeStartElement("bpmn2", "definitions", "http://www.omg.org/spec/BPMN/20100524/MODEL");
        xtw.setDefaultNamespace("http://www.omg.org/spec/BPMN/20100524/MODEL");
        xtw.writeDefaultNamespace("http://www.omg.org/spec/BPMN/20100524/MODEL");
        xtw.writeNamespace("bpmn2", "http://www.omg.org/spec/BPMN/20100524/MODEL");
        xtw.writeNamespace("xsi", "http://www.w3.org/2001/XMLSchema-instance");
        xtw.writeNamespace("xsd", "http://www.w3.org/2001/XMLSchema");
        xtw.writeNamespace("activiti", "http://activiti.org/bpmn");
        xtw.writeNamespace("bpmndi", "http://www.omg.org/spec/BPMN/20100524/DI");
        xtw.writeNamespace("omgdc", "http://www.omg.org/spec/DD/20100524/DC");
        xtw.writeNamespace("omgdi", "http://www.omg.org/spec/DD/20100524/DI");
        Iterator var3 = model.getNamespaces().keySet().iterator();
```
